tests/foreman/cli/test_docker.py

```diff
@@ -180,21 +230,19 @@ def test_update_docker_repo_name(self, new_name):
 
         @Feature: Docker
 
-        @Status: Manual
-
         """
+        repo = _make_docker_repo(
+            make_product({'organization-id': self.org_id})['id'])
+        result = Repository.update({
```

Comment:
It will be nice to have update procedure verification in a way that before update you have one name and another after.